packages/hint/docs/user-guide/concepts/hints.md

```diff
@@ -1,70 +1,78 @@
+---
+date: 08/27/2020
+---
 # Hints
 
-A hint is a test that your website needs to pass. Webhint comes with
-several [built in ones][hints], but you can create your own or download
-them from `npm`. You can read more about [how to create hints in the
-contributor guide][how to hint].
+A hint is a test that your website needs to pass.  `webhint` includes several [built-in hints][HintsIndex], but you may create your own or download more hints from `npm`.  For more information about how to create hints, go to the [contributor guide][ContributorGuideHowToHint].
 
 ## Installing hints
 
-To utilize a hint, install any package matching `@hint/hint-`,
-`webhint-hint-`, or `@scope/webhint-hint-`. Then, add that package's
-name to your .hintrc's `hints` array or object. Packages within the
-`@hint/` namespace (like, for example, `@hint/hint-html-checker`) can be
-added using their short name.
+Complete the following actions to utilize a hint.
 
-For example, to use the [Nu HTML test][html-checker] first install its
-package:
+1.  Search for a `webhint` package that begins with one of the following statements.
+    *   `@hint/hint-`
+    *   `webhint-hint-`
+    *   `@scope/webhint-hint-`
+1.  Install your selected package.
+1.  Add the name of that package to the `hints` array or object in your `.hintrc` file.
 
-```bash
-npm i -D @hint/hint-html-checker
-```
+    > [!NOTE]
+    > Packages within the `@hint/` namespace, such as `@hint/hint-html-checker`, may be added using a short name. The short name does not include the `webhint` package reserved text. Example `@hint/hint-html-checker` is shortened to `html-checker`.
```

Comment:
@molant @antross
Please confirm that the relationship between package name and short name is the same for all `webhint` packages. 
*   `@hint/hint-` 
*   `webhint-hint-`  
*   `@scope/webhint-hint-`

Reply:
@molant @antross
Is there an update?